parsers/nmparser/structs.go

```diff
@@ -31,7 +31,14 @@ type ParametersData struct {
 	Fixed            ParametersResult   `json:"fixed,omitempty"`
 }
 
+// RunHeuristics ...
+// some values are defined as pointers to support tri-state: true, false, nil
 type RunHeuristics struct {
+	CovarianceStepOmitted *bool `json:"covariance_step_omitted,omitempty"`
```

Comment:
why pointers? there doesn't seem to be any reason for pointers and breaks the consistency of the rest of the struct api